lib/helpers.js

```diff
@@ -224,6 +225,21 @@ exports.verifyDelegation = async ({keystoreId, host, capability}) => {
   return results;
 };
 
+exports.verifyHeaderValue = async (req, res, next) => {
+  const expectedDigest = req.headers.digest;
+  const {verified} = await verifyHeaderValue({
+    data: req.body, headerValue: expectedDigest});
```

Comment:
I'm not sure this will work all the time. What if the body was sent with different whitespace? I suspect we need to be computing the digest on the data before it is parsed rather than using `req.body`. Once we figure out how to do this in a consistent way, we should make it an express middleware we can use in multiple places.

cc: @davidlehn -- where are we with being able to configure per-route special body parsing/processing middleware (so we can override the defaults)?

Reply:
I'm filing an issue about this to improve support in the future, we shouldn't hold up the PR for this.

Reply:
Filed #37.

Reply:
Good point, re verifying the digest on the data. 
We could use the technique described in this post: https://flaviocopes.com/express-get-raw-body/ (basically preserving a copy of the rawBody of the request, in addition to the parsed).

Reply:
Ah, n/m, the problem is that we're parsing the request on the Bedrock level, before we even get to this route definition, right?